lib/services/MessageQueue.js

```diff
@@ -4,24 +4,13 @@
  * @description the services for connecting message queue
  */
 
-let _sharedMessageQueue = null
-let _messageQueueLib = null
-
 class MessageQueue {
-
-  static get messageQueueLib() {
-
-    if (!_messageQueueLib) {
-
-      _messageQueueLib = require('amqplib')
-
-    }
-
-    return _messageQueueLib
-
+  constructor(config) {
+    this.init(config)
```

Comment:
just rename `init` to `constructor`?

Reply:
there are some cases that we need to separate the function. I can't deny that this case seems a bit redundant. Let refactor all the constructor and init function. after these PRs.

Reply:
it's weird that an object can init more than once 🤔